docs/kratos/concepts/security.mdx

```diff
@@ -3,210 +3,40 @@ id: security
-          - hook: session
-```
-
-## Account Takeover Defenses
-
-The Settings Flow implements account takeover defenses as it isn't possible to change the password without knowing the existing
-password. A good example of this flow is the
-[GitHub sudo mode](https://help.github.com/en/github/authenticating-to-github/sudo-mode).
-
-## Passwords
-
-Head over to the [password policy page](../../concepts/password-policy.mdx).
-
-## Anti-automation
-
-:::warning
-
-This feature is a work in progress and is tracked as [kratos#138](https://github.com/ory/kratos/issues/138).
-
-:::
-
-Actions that cause out-of-band communications, such as sending an activation link via email or an activation code via SMS, can be
-abused by automated systems. The goal of such an attack is to send out so many emails or SMS, that your reputation worsens (spam
-filters) or you're faced with massive costs (carrier fees).
-
-CAPTCHA renders these attacks either very difficult or impossible. CAPTCHA will be required in the following scenarios:
-
-- The user tries to register more than one account within 72 hours.
-- The user failed provide valid credentials for the third time within 12 hours.
-- The user tries to recover their account for the second time within 72 hours.
-
-For integration guidelines, please check the individual flow's (registration, login, account recovery) integration documentation.
-
-## Bruteforce attacks
-
-Will be addressed in a future release.
-
-## Phishing attacks
-
-Will be addressed in a future release.
-
-## Social engineering attacks
-
-Will be addressed in a future release.
-
-## SMS spoofing attacks
+## Digital identity guidelines
 
-Will be addressed in a future release.
+There is no universally accepted standard for digital identity.  
+Ory follows [Digital Identity Guidelines](https://pages.nist.gov/800-63-3/) established by the
+[National Institute of Standards and Technology (NIST)](https://www.nist.gov/).  
+These guidelines are accompanied by a [FAQ](https://pages.nist.gov/800-63-3/) that provides additional information and answers to
+common questions.
 
-## Account enumeration
+## Brute-force defenses
 
-:::warning
+Ory Network provides Ory Identities with protection against brute force attacks by rate-limiting requests to API public endpoints,
+for example login and registration endpoints.
 
-This feature is a work in progress and is tracked as [kratos#133](https://github.com/ory/kratos/issues/133).
+When self-hosting the Ory Kratos Identity Server, it is the responsibility of the administrator to implement and manage
+rate-limiting or other measures to ensure the security of the network.
 
-It doesn't yet work as documented!
+## Account takeover defenses
 
-:::
+The [settings flow](../self-service/flows/user-settings) implements account takeover defenses by requiring the current password
+before allowing the user to change it. This makes it impossible for an attacker to change the password without knowing the
+existing password.
```

Comment:
Technically changing passwords is possible only when users have a privileged session. To get a privileged session, users must authenticate. Do you think that it's a good idea to describe this?